{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "734210f5-8199-4915-a36b-7fd7e280503e",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2022-02-06 16:00:58,651 [worker_pid=77] Bot: onlymemes [@onlymemebot]\n",
      "2022-02-06 16:00:59,857 [worker_pid=77] Updates were skipped successfully.\n",
      "2022-02-06 16:00:59,857 [worker_pid=77] Start polling.\n",
      "2022-02-06 16:02:34,147 [worker_pid=77] Received inline query [ID:1187924785364615132] from user [ID:276585292]\n",
      "2022-02-06 16:02:37,977 [worker_pid=77] Received inline query [ID:1187924786456325242] from user [ID:276585292]\n",
      "{'update': '{\"id\": \"1187924786456325242\", \"from\": {\"id\": 276585292, \"is_bot\": false, \"first_name\": \"Daniel\", \"last_name\": \"Kovalenko\", \"username\": \"daniel_kovalenko\", \"language_code\": \"en\"}, \"chat_type\": \"private\", \"query\": \"qwert\", \"offset\": \"\"}', 'template_options': [\"(image['template_name']+'__'+str(uuid4()))[:63]\"]}\n",
      "2022-02-06 16:02:37,978 [worker_pid=77] Process update [ID:734871206]: [success] (in 1 ms)\n",
      "2022-02-06 16:02:45,441 [worker_pid=77] Received inline query [ID:1187924787343302040] from user [ID:276585292]\n",
      "2022-02-06 16:02:47,545 [worker_pid=77] Received inline query [ID:1187924783937398119] from user [ID:276585292]\n",
      "{'update': '{\"id\": \"1187924783937398119\", \"from\": {\"id\": 276585292, \"is_bot\": false, \"first_name\": \"Daniel\", \"last_name\": \"Kovalenko\", \"username\": \"daniel_kovalenko\", \"language_code\": \"en\"}, \"chat_type\": \"private\", \"query\": \"qwerty\", \"offset\": \"\"}', 'template_options': [\"(image['template_name']+'__'+str(uuid4()))[:63]\"]}\n",
      "2022-02-06 16:02:47,546 [worker_pid=77] Process update [ID:734871208]: [success] (in 1 ms)\n",
      "2022-02-06 16:03:26,171 [worker_pid=77] Received inline query [ID:1187924787785075663] from user [ID:276585292]\n",
      "2022-02-06 16:03:27,507 [worker_pid=77] Received inline query [ID:1187924784316010188] from user [ID:276585292]\n",
      "{'update': '{\"id\": \"1187924784316010188\", \"from\": {\"id\": 276585292, \"is_bot\": false, \"first_name\": \"Daniel\", \"last_name\": \"Kovalenko\", \"username\": \"daniel_kovalenko\", \"language_code\": \"en\"}, \"chat_type\": \"private\", \"query\": \"q\", \"offset\": \"\"}', 'template_options': [\"(image['template_name']+'__'+str(uuid4()))[:63]\"]}\n",
      "2022-02-06 16:03:27,507 [worker_pid=77] Process update [ID:734871210]: [success] (in 1 ms)\n",
      "2022-02-06 16:03:28,925 [worker_pid=77] Received inline query [ID:1187924784903401948] from user [ID:276585292]\n",
      "{'update': '{\"id\": \"1187924784903401948\", \"from\": {\"id\": 276585292, \"is_bot\": false, \"first_name\": \"Daniel\", \"last_name\": \"Kovalenko\", \"username\": \"daniel_kovalenko\", \"language_code\": \"en\"}, \"chat_type\": \"private\", \"query\": \"qwert\", \"offset\": \"\"}', 'template_options': [\"(image['template_name']+'__'+str(uuid4()))[:63]\"]}\n",
      "2022-02-06 16:03:45,444 [worker_pid=77] Received inline query [ID:1187924786152625101] from user [ID:276585292]\n",
      "2022-02-06 16:03:46,054 [worker_pid=77] Received inline query [ID:1187924787510910408] from user [ID:276585292]\n",
      "2022-02-06 16:03:47,804 [worker_pid=77] Received inline query [ID:1187924786854643859] from user [ID:276585292]\n",
      "{'update': '{\"id\": \"1187924786854643859\", \"from\": {\"id\": 276585292, \"is_bot\": false, \"first_name\": \"Daniel\", \"last_name\": \"Kovalenko\", \"username\": \"daniel_kovalenko\", \"language_code\": \"en\"}, \"chat_type\": \"private\", \"query\": \"ghgj\", \"offset\": \"\"}', 'template_options': [\"(image['template_name']+'__'+str(uuid4()))[:63]\"]}\n",
      "2022-02-06 16:03:47,804 [worker_pid=77] Process update [ID:734871214]: [success] (in 1 ms)\n",
      "2022-02-06 16:04:35,667 [worker_pid=77] Cause exception while getting updates.\n",
      "Traceback (most recent call last):\n",
      "  File \"/opt/conda/lib/python3.7/site-packages/aiogram/dispatcher/dispatcher.py\", line 384, in start_polling\n",
      "    allowed_updates=allowed_updates\n",
      "  File \"/opt/conda/lib/python3.7/site-packages/aiogram/bot/bot.py\", line 97, in get_updates\n",
      "    result = await self.request(api.Methods.GET_UPDATES, payload)\n",
      "  File \"/opt/conda/lib/python3.7/site-packages/aiogram/bot/base.py\", line 209, in request\n",
      "    proxy=self.proxy, proxy_auth=self.proxy_auth, timeout=self.timeout, **kwargs)\n",
      "  File \"/opt/conda/lib/python3.7/site-packages/aiogram/bot/api.py\", line 140, in make_request\n",
      "    return check_result(method, response.content_type, response.status, await response.text())\n",
      "  File \"/opt/conda/lib/python3.7/site-packages/aiogram/bot/api.py\", line 119, in check_result\n",
      "    exceptions.ConflictError.detect(description)\n",
      "  File \"/opt/conda/lib/python3.7/site-packages/aiogram/utils/exceptions.py\", line 140, in detect\n",
      "    raise err(cls.text or description)\n",
      "aiogram.utils.exceptions.TerminatedByOtherGetUpdates: Terminated by other getupdates request; make sure that only one bot instance is running\n",
      "2022-02-06 16:04:45,952 [worker_pid=77] Cause exception while getting updates.\n",
      "Traceback (most recent call last):\n",
      "  File \"/opt/conda/lib/python3.7/site-packages/aiogram/dispatcher/dispatcher.py\", line 384, in start_polling\n",
      "    allowed_updates=allowed_updates\n",
      "  File \"/opt/conda/lib/python3.7/site-packages/aiogram/bot/bot.py\", line 97, in get_updates\n",
      "    result = await self.request(api.Methods.GET_UPDATES, payload)\n",
      "  File \"/opt/conda/lib/python3.7/site-packages/aiogram/bot/base.py\", line 209, in request\n",
      "    proxy=self.proxy, proxy_auth=self.proxy_auth, timeout=self.timeout, **kwargs)\n",
      "  File \"/opt/conda/lib/python3.7/site-packages/aiogram/bot/api.py\", line 140, in make_request\n",
      "    return check_result(method, response.content_type, response.status, await response.text())\n",
      "  File \"/opt/conda/lib/python3.7/site-packages/aiogram/bot/api.py\", line 119, in check_result\n",
      "    exceptions.ConflictError.detect(description)\n",
      "  File \"/opt/conda/lib/python3.7/site-packages/aiogram/utils/exceptions.py\", line 140, in detect\n",
      "    raise err(cls.text or description)\n",
      "aiogram.utils.exceptions.TerminatedByOtherGetUpdates: Terminated by other getupdates request; make sure that only one bot instance is running\n",
      "2022-02-06 16:04:51,332 [worker_pid=77] Received inline query [ID:1187924786574665376] from user [ID:276585292]\n",
      "{'update': '{\"id\": \"1187924786574665376\", \"from\": {\"id\": 276585292, \"is_bot\": false, \"first_name\": \"Daniel\", \"last_name\": \"Kovalenko\", \"username\": \"daniel_kovalenko\", \"language_code\": \"en\"}, \"chat_type\": \"private\", \"query\": \"qwe\", \"offset\": \"\"}', 'template_options': [\"(image['template_name']+'__'+str(uuid4()))[:63]\"]}\n",
      "2022-02-06 16:04:51,333 [worker_pid=77] Process update [ID:734871217]: [success] (in 1 ms)\n",
      "2022-02-06 16:04:56,170 [worker_pid=77] Cause exception while getting updates.\n",
      "Traceback (most recent call last):\n",
      "  File \"/opt/conda/lib/python3.7/site-packages/aiogram/dispatcher/dispatcher.py\", line 384, in start_polling\n",
      "    allowed_updates=allowed_updates\n",
      "  File \"/opt/conda/lib/python3.7/site-packages/aiogram/bot/bot.py\", line 97, in get_updates\n",
      "    result = await self.request(api.Methods.GET_UPDATES, payload)\n",
      "  File \"/opt/conda/lib/python3.7/site-packages/aiogram/bot/base.py\", line 209, in request\n",
      "    proxy=self.proxy, proxy_auth=self.proxy_auth, timeout=self.timeout, **kwargs)\n",
      "  File \"/opt/conda/lib/python3.7/site-packages/aiogram/bot/api.py\", line 140, in make_request\n",
      "    return check_result(method, response.content_type, response.status, await response.text())\n",
      "  File \"/opt/conda/lib/python3.7/site-packages/aiogram/bot/api.py\", line 119, in check_result\n",
      "    exceptions.ConflictError.detect(description)\n",
      "  File \"/opt/conda/lib/python3.7/site-packages/aiogram/utils/exceptions.py\", line 140, in detect\n",
      "    raise err(cls.text or description)\n",
      "aiogram.utils.exceptions.TerminatedByOtherGetUpdates: Terminated by other getupdates request; make sure that only one bot instance is running\n",
      "2022-02-06 16:05:05,834 [worker_pid=77] Cause exception while getting updates.\n",
      "Traceback (most recent call last):\n",
      "  File \"/opt/conda/lib/python3.7/site-packages/aiogram/dispatcher/dispatcher.py\", line 384, in start_polling\n",
      "    allowed_updates=allowed_updates\n",
      "  File \"/opt/conda/lib/python3.7/site-packages/aiogram/bot/bot.py\", line 97, in get_updates\n",
      "    result = await self.request(api.Methods.GET_UPDATES, payload)\n",
      "  File \"/opt/conda/lib/python3.7/site-packages/aiogram/bot/base.py\", line 209, in request\n",
      "    proxy=self.proxy, proxy_auth=self.proxy_auth, timeout=self.timeout, **kwargs)\n",
      "  File \"/opt/conda/lib/python3.7/site-packages/aiogram/bot/api.py\", line 140, in make_request\n",
      "    return check_result(method, response.content_type, response.status, await response.text())\n",
      "  File \"/opt/conda/lib/python3.7/site-packages/aiogram/bot/api.py\", line 119, in check_result\n",
      "    exceptions.ConflictError.detect(description)\n",
      "  File \"/opt/conda/lib/python3.7/site-packages/aiogram/utils/exceptions.py\", line 140, in detect\n",
      "    raise err(cls.text or description)\n",
      "aiogram.utils.exceptions.TerminatedByOtherGetUpdates: Terminated by other getupdates request; make sure that only one bot instance is running\n"
     ]
    }
   ],
   "source": [
    "!python /app/apps/bot/main.py"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.11"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
